.github/workflows/test-and-deploy.yml

```diff
@@ -11,10 +11,10 @@ jobs:
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
-      - name: Use Node.js 20.x
+      - name: Use Node.js 22.x
         uses: actions/setup-node@v4
         with:
-          node-version: 20.x
+          node-version: 22.x
```

Comment:
Should this (and the other .x versions) also be pinned?

Reply:
This one is just for ts-check and linting etc, I think it's ok?

Reply:
Ah yeah you are right